arrow-select/src/take.rs

```diff
@@ -1985,6 +1987,23 @@ mod tests {
         assert_eq!(&values, &[Some(23), Some(4), None, None])
     }
 
+    #[test]
+    fn test_take_fixed_size_list_null_indices() {
+        let indices = Int32Array::new(vec![0, 1].into(), Some(NullBuffer::from(vec![true, false])));
```

Comment:
nit: maybe more straight forward to not separate values and nullability:
```suggestion
        let indices = Int32Array::from_iter([Some(1), None]);
```